core/no_certificate_validation.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+use rustls::ServerCertVerifier;
+use rustls::RootCertStore;
+use rustls::Certificate;
+use rustls::ServerCertVerified;
+use rustls::TLSError;
+use rustls::HandshakeSignatureValid;
+use rustls::internal::msgs::handshake::DigitallySignedStruct;
+use webpki::DNSNameRef;
+
+pub struct NoCertificateVerification {
+  excluded: Vec<String>
+}
+
+impl NoCertificateVerification {
+  pub fn new(excluded: Vec<String>) -> Self {
+    Self {
+      excluded
+    }
+  }
+}
+
+impl ServerCertVerifier for NoCertificateVerification {
+  fn verify_server_cert(&self,
+                        _roots: &RootCertStore,
+                        _presented_certs: &[Certificate],
+                        dns_name: DNSNameRef<'_>,
+                        _ocsp: &[u8]) -> Result<ServerCertVerified, TLSError> {
+      fn convert_to_string<T: AsRef<str>>(s: T) -> String {
+        s.as_ref().to_owned()
+      }
+      let dns_name: String = convert_to_string(dns_name.to_owned());
```

Comment:
Somehow I could not do `dns_name.to_owned().as_ref().to_owned()` and cast it to String, that's why the `convert_to_string` is present - any better ideas?